vck-aries/src/commonMain/kotlin/at/asitplus/wallet/lib/aries/PresentProofProtocol.kt

```diff
@@ -215,7 +215,6 @@ class PresentProofProtocol(
             inputDescriptors = listOf(
                 InputDescriptor(
                     name = credentialScheme.vcType!!,
```

Comment:
Why did the `schema` go away?

Reply:
as briefly discussed in the last pull request we used [DIF version 1.0.0](https://identity.foundation/presentation-exchange/spec/v1.0.0/#input-descriptor) to define this class. Beginning with [DIF version 2.0.0](https://identity.foundation/presentation-exchange/spec/v2.0.0/#input-descriptor) `schema` is no longer a member. 
I became aware of this in my last pull request since this broke interoperability with current implementations when actively used - compare for instance [EUDI verifier](https://verifier.eudiw.dev/home) - so i removed it.
As far as i know we did not actively use it anyway right?

Reply:
#109

Reply:
also #110